shared/actions/chat2/index.js

```diff
@@ -38,7 +38,7 @@ const inboxRefresh = (
 ) => {
   const username = state.config.username || ''
 
-  const onUnverified = function({inbox}: RPCChatTypes.ChatUiChatInboxUnverifiedRpcParam) {
+  const onUnverified = function({inbox}) {
```

Comment:
All these types went away as it actually is typed now. Before it was `any` and you'd have to say what the type was yourself